src/components/Layout/UsersTable/UsersTable.tsx

```diff
@@ -0,0 +1,62 @@
+import React from 'react'
+import MaterialTable from 'material-table'
+
+export default function MaterialTableDemo() {
+  const [state, setState] = React.useState({
+    columns: [
+      { title: 'Name', field: 'name' },
+      { title: 'Surname', field: 'surname' },
+      { title: 'Birth Year', field: 'birthYear' },
+      {
+        title: 'Birth Place',
+        field: 'birthCity'
+      }
+    ],
+    data: [
```

Comment:
Sooner or later u will be getting the data from the BackEnd so I encourage you to replace this second part of the object `data` with a prop coming from the parent component (in this case `src/pages/backOffice/Users/Users.tsx`) and also u will have to add the corresponding typings for it.


I show u an example for this case but in the future I will ask u to add them yourself:

in the types.ts that is located in the root of this component set:
```
interface User {
    name: string
    surname: string
    birthYear: number
    birthCity: number
}

export interface ComponentProps {
    users: User[]
}
```
import that in this file: `import { ComponentProps } from './types'` and then use it in the component

`data: props.users`